pkg/kubelet/server/server.go

```diff
@@ -167,7 +167,7 @@ func ListenAndServeKubeletServer(
 
 // ListenAndServeKubeletReadOnlyServer initializes a server to respond to HTTP network requests on the Kubelet.
 func ListenAndServeKubeletReadOnlyServer(host HostInterface, resourceAnalyzer stats.ResourceAnalyzer, address net.IP, port uint, enableCAdvisorJSONEndpoints bool) {
-	klog.V(1).Infof("Starting to listen read-only on %s:%d", address, port)
+	klog.V(1).InfoS("Starting to listen read-only server", "address", address, "port", port)
```

Comment:
```suggestion
	klog.V(1).InfoS("Starting to listen read-only", "address", address, "port", port)
```

Reply:
Does it make any sense that this is V(1) and the previous server is always logged? Maybe we should just always log both...

Reply:
Thanks for the advice, I have fixed it. I also think V(1) is redundant, I've deleted it.